packages/ketcher-react/src/infrastructure/services/struct/remoteStructServiceProvider.ts

```diff
@@ -0,0 +1,17 @@
+import { StructService, StructServiceProvider } from './structService.types'
+import IndigoService from './remoteStructService'
+
+class RemoteStructServiceProvider implements StructServiceProvider {
+  private baseUrl!: string
+
+  constructor(baseUrl: string) {
+    this.baseUrl = !baseUrl || /\/$/.test(baseUrl) ? baseUrl : baseUrl + '/';
+  }
+
+  initStructService(options: any): StructService {
+
+    ;(global as any).ketcher.apiPath = this.baseUrl
```

Comment:
It should not be a part of StructServiceProvider

Reply:
reverted back